ynr/apps/sopn_parsing/tests/__init__.py

```diff
@@ -5,3 +5,12 @@ def should_skip_pdf_tests():
         return False
     except ImportError:
         return True
+
+
+def should_skip_conversion_tests():
+    try:
+        import pandoc  # noqa
```

Comment:
Should this be `pypandoc`? May be why the tests are not running (if I try to import `pandoc` in a shell I get an error)

Reply:
Yes! It should. Good catch.